integration_tests/features/WalletCli.feature

```diff
@@ -150,3 +151,14 @@ Feature: Wallet CLI
         And I create committee definition for asset on wallet WALLET via command line
         And mining node MINE mines 1 blocks
         Then WALLET is connected to BASE
+
+     Scenario: As a user I want to publish a contract definition via command line
```

Comment:
Should we tag this scenario with something for like @layertwo or @l2ud?. Something so we can scope or run specifically the use case defined specs for layer 2 work. Not needed for every new spec but for the ones where we're obviously completing a defined user scenario.

Reply:
Good idea, maybe `@dan_layer`? Just to keep consistent naming across the project